### PR TITLE
feat : export metrics data ( opentelemetry  + Prometheus)

### PR DESCRIPTION
**Pull request by Nagaprasadvr**
1. Export opentelemetry and Prometheus metrics data 
2. Setup docker config for running opentelemetry and Prometheus services which will pull this metrics data and process it

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [workspace]
 members = ["crates/*"]
-exclude = ["crates/test", "crates/test-streams"]
+exclude = ["crates/demo", "crates/test-streams"]
```
Comment (kespinola) [inline Cargo.toml]: @ray-kast what do you think about have our examples in `examples`?

```
examples/
  simple
  streams
crates/
  core
  runtime
```

```suggestion
exclude = ["examples/*"]
```

Comment (Nagaprasadvr) [reply Cargo.toml:3]: yeah this is ideal makes sense

Comment (kespinola) [reply Cargo.toml]: @Nagaprasadvr make this change please

Comment (Nagaprasadvr) [reply Cargo.toml]: Will do

**Comment by Nagaprasadvr** [comment]
#15 check this

**Pull request closed without merging**